lib/Conversion/GPUToNVVM/CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
+include_directories(../GPUCommon)
```

Comment:
It would be preferable to avoid `include_directories` to make it easier to integrate this with bazil builds. Instead, please use relative paths in the `#include` directive.

Reply:
pushed out a new commit with the requested change